bin/common/src/metabuild_common/files.clj

```diff
@@ -130,3 +130,12 @@
   "Get the size, in bytes, of the file at `path`."
   ^Long [^String path]
   (Files/size (nio-path path)))
+
```

Comment:
thought would need this originally when the plan was to just upload to s3. Now that the plan is just to stick these in the jar this is unnecessary but seemed helpful. It's true that it is just a thin wrapper around interop, but now there's autocompletion and docstrings. Happy to remove if this isn't needed.